README.md

```diff
@@ -0,0 +1,86 @@
+# Fewlines Readline
+
+This package uses the [NodeJs Readline](https://nodejs.org/api/readline.html) and wrap it in a custom package in order to simplify its use and testing with jest mocks in Typescript.
+
+## Usage
+
+`fwl-readline` is not yet available on npm.
+To install it, add this in your `package.json` file:
+
+### Installation
+
+```json
+"dependencies": {
+  "fwl-readline": "fewlines-education/fwl-readline#master"
+}
+```
+
+### Import
+```js
+// example.js
+import * as readline from "fwl-readline";
+
+```
+
+### Create a reader
+
+We decided to simplify the initial API of the readline package. For this reason, we only export the `createInterface` function.
+
+This functions takes an `object` parameter with to keys:
+- `input`: generally the `process.stdin`
+- `output`: generally the `process.stdout`
+
+```js
+export const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+```
+
+### Type
+
+In the example above, the created `rl` will have the `Reader` type:
+
+```ts
+interface Reader {
+  question: (query: string, callback: (answer: string) => void) => void;
+  close: () => void;
+}
+```
+
+### Use the created reader
```

Comment:
⛏️ `### Usage`

Reply:
I already have the `## Usage` in fact.
But in the `##` we need to show how to `### Use the created reader` 😇

Reply:
Woops my bad 🙇.